spring-cloud-gcp-autoconfigure/src/main/resources/org/springframework/cloud/gcp/autoconfigure/logging/logback-json-appender.xml

```diff
@@ -3,7 +3,7 @@
 Stackdriver json-log format provided for import.
 -->
 <included>
-    <property name="projectId" value="${projectId:-${GOOGLE_CLOUD_PROJECT}}"/>
+    <property name="projectId" value="${projectId:-${SPRING_CLOUD_GCP_LOGGING_PROJECT_ID}}"/>
```

Comment:
IIUC this aims at grabbing the value of the `spring.cloud.gcp.logging.project-id` property?
What if an application configured the project ID as `spring.cloud.gcp.project-id`?

Reply:
That won't be used. They'll need to manually add <springProperty> to reference to a project ID.

GOOGLE_CLOUD_PROJECT is actually not explicit enough since it could override a lot of things.  It's actually possible to send a multi-project trace in Stackdriver. I.e., traces from multiple projects show up in the same Trace view, but the log links to the originating project.